### DISCUSSION TITLE
Feature Request (Github Actions): Branches pattern with wildcard in the search bar of the web UI

### DISCUSSION THREAD

**Discussion by trash-anger** [community/community#62409] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi,

When I go to `https://github.com/:org/:repo/actions` I'd like to have a way to filter branches with a wildcard (e.g. `branch:release*` or `branch:feature*`).

I'm surprised that I haven't found a way to do it... Is there a way to do that already?

<img width="301" alt="image" src="https://github.com/community/community/assets/7171011/066af75f-210c-448c-85e9-f65279ecf062">

**Comment by trash-anger** [comment]
Up!

**Reply by mnrkholis** [reply]
hello brodi

**Comment by diegocatao** [comment]
Hi @trash-anger,
Sorry, GH Actions currently does not support wildcards on that input.

**Reply by trash-anger** [reply]
Is there a plan to support that? 

That should be super easy to implement and right now, having a templatized pipeline that run on all branches is a mess because there's no way to retrieve the execution on a set of multiple branches organized (e.g. feature/* hotfix/* release/*).

**Reply by trash-anger** [reply]
Up!

**Reply by trash-anger** [reply]
Hi @diegocatao,
Do you know if there's an ETA for that? It seems quitte easy to implement and it would be super valuable! Don't you think?

**Reply by mnrkholis** [reply]
you cool

**Comment by roger-marley-lrn** [comment]
+1 that this would be very useful. 

Either making the `head:HEAD_BRANCH` and `base:BASE_BRANCH` take a wildcard syntax, or extending the `in` qualifier so it can search in those branch names (in addition to title/body/comments).
